### PR TITLE
[M68K] store correct register value in op.reg_pair

### LINKED ISSUE #1406: m68k: reg_pair off by one
If we modify `test_m68k.c` to print out the `M68K_OP_REG_PAIR` case:

~~~diff
diff --git a/tests/test_m68k.c b/tests/test_m68k.c
index 9eb37104..df6cef7b 100644
--- a/tests/test_m68k.c
+++ b/tests/test_m68k.c
@@ -137,7 +137,12 @@ static void print_insn_detail(cs_insn *ins)
                                break;
                        case M68K_OP_REG_BITS:
                                printf("\t\toperands[%u].type: REG_BITS = $%x\n", i, op->register_bits);
-
+                               break;
+                       case M68K_OP_REG_PAIR:
+                               printf("\t\toperands[%u].type: REG_PAIR = (%s, %s)\n", i,
+                                       cs_reg_name(handle, op->reg_pair.reg_0),
+                                       cs_reg_name(handle, op->reg_pair.reg_1));
+                               break;
                }
        }

~~~

Then we can see an off-by-one error (`d3, d4` instead of `d4, d5`):

~~~
0x1020: mulu.l  d0, d4:d5
        op_count: 2
        reading from reg: d0
        writing to reg:   d4
        writing to reg:   d5
        groups_count: 0
                operands[0].type: REG = d0
                operands[1].type: REG_PAIR = (d3, d4)
~~~

We can see the error in how the internal instruction printing works:

https://github.com/aquynh/capstone/blob/40b8bfa39788c8e99b01a542ebe43ef582f807dc/arch/M68K/M68KInstPrinter.c#L125-L129

The actual `m68k_reg` value should be stored, not `reg - M68K_REG_D0`.

### PR DESCRIPTION
**Pull request by tmfink**
Fixes #1406 

Also fixes a small type in the licensing text.

### CODE REVIEW

File: arch/M68K/M68KDisassembler.c [hunk 1]
```diff
@@ -27,7 +27,7 @@
  * THE SOFTWARE.
  */
 
-/* The code bellow is based on MUSASHI but has been heavily modified for Capstone by
+/* The code below is based on MUSASHI but has been heavily modified for capstone by
  * Daniel Collin <daniel@collin.com> 2015-2019 */
```
Comment (aquynh) [inline arch/M68K/M68KDisassembler.c]: why lowercase for "capstone" ;-?

**Comment by aquynh** [comment]
@emoon please ack.

File: arch/M68K/M68KDisassembler.c (hunk 1 above)
Comment (tmfink) [reply arch/M68K/M68KDisassembler.c]: That was an accident...

**Comment by emoon** [comment]
Looks good! Thanks for the fix

**Comment by aquynh** [comment]
merged, thanks!

**Comment by aquynh** [comment]
cherry-picked for "v4" branch, too.

**Pull request merged**